product-service/handlers/getProductsList.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+import data from '../data/mock.json';
+
+export const getProductsList = async event => {
+
+  try{
+    if(!data){
+      throw new Error( 'Error: products not found1!' );
+    }
+
+    return {
+        statusCode: 200,
+        headers: {
+          'Access-Control-Allow-Origin': '*',
+          'Access-Control-Allow-Credentials': true,
+        },
```

Comment:
you could create a util to process response and doesn't write headers and status every time, like:
```
 data =>   ({
       statusCode: 200,
        headers: {
          'Access-Control-Allow-Origin': '*',
          'Access-Control-Allow-Credentials': true,
        },
        body: JSON.stringify(data)
    })
```